src/algorithms.jl

```diff
@@ -24,6 +36,17 @@ See [`IntervalOptimisation.jl`](https://github.com/JuliaIntervals/IntervalOptimi
 ### Notes
 
 To use this algorithm, you need to import `IntervalOptimisation.jl`
+
+### Examples
+
+```jldoctest
+julia> using IntervalOptimisation
+
+julia> enclose(x -> 1 - x^4 + x^5, 0..1, MooreSkelboeEnclosure()) # default values
```

Comment:
```suggestion
julia> enclose(x -> 1 - x^4 + x^5, 0..1, MooreSkelboeEnclosure()) # default tolerance
```

Reply:
but the tolerance is not the only parameter you can set, you can also tune the search structure

Reply:
here "values" may be too general; `default algorithm parameters` or `default tolerance` is better imo

Reply:
yes values is not a very descriptive word, what about `default parameters`?

Reply:
ok fair enough, i updated my suggestion